daffodil-test/src/test/scala/org/apache/daffodil/section15/choice_groups/TestSharedGroups.scala

```diff
@@ -35,6 +35,6 @@ class TestSharedGroups {
 
   @Test def testShowsSharedGroupOkFirstUse() = { runner.runOneTest("Ack") }
   // DAFFODIL-2615 - shared groups problem
-  //@Test def testShowsSharedGroupFailsSecondUse() = { runner.runOneTest("Nack") }
+  @Test def testShowsSharedGroupFailsSecondUse() = { runner.runOneTest("Nack") }
```

Comment:
Is there any value in adding a negative test with a bad infoset like this:

```
          <messageID>2</messageID>
          <nack>X</nack>
          <ackInfo>A</ackInfo>
```

So we mix nack and ackinfo, so we ensure that the correct map is used and not some combined map or something where both nackInfo and ackInfo are accepted?